lib/features/children/add_member/pages/add_member_form_page.dart

```diff
@@ -0,0 +1,346 @@
+            icon: Icon(FontAwesomeIcons.circlePlus,
+                size: 32,
+                color: (_allowanceController > 998)
+                    ? Colors.grey
+                    : Theme.of(context).colorScheme.primary),
+          ),
+        ],
+      );
+
+  Widget createChildForm(GlobalKey<FormState> formKey, String currency) => Form(
+        key: formKey,
+        child: Column(
+          mainAxisSize: MainAxisSize.min,
+          children: [
+            FamilyTextFormField(
+              validator: (value) {
+                if (value == null || value.isEmpty) {
+                  return "Please enter the child's name";
+                }
+                if (value.length < 2) {
+                  return 'Please enter a valid name';
+                }
+                if (value.length > 20) {
+                  return 'Name is too long';
+                }
+                return null;
+              },
+              controller: _nameController,
+              hintText: context.l10n.firstName,
+              keyboardType: TextInputType.name,
+            ),
+            FamilyTextFormField(
+              validator: (value) {
+                if (value == null || value.isEmpty) {
+                  return "Please enter the child's age";
+                }
+                if (int.tryParse(value) == null) {
+                  return 'Please enter a valid age';
+                }
+                if (int.parse(value) > 18) {
+                  return 'Please add an adult instead';
+                }
+                return null;
+              },
+              controller: _ageController,
+              hintText: 'Age',
+              keyboardType: TextInputType.number,
+            ),
+            const GivingAllowanceInfoButton(),
+            allowanceCounter(currency),
+          ],
+        ),
+      );
+
+  Widget setUpFamilyHeader() => RichText(
+        textAlign: TextAlign.center,
+        text: TextSpan(
+          text: 'Set up Family\n',
+          style: Theme.of(context).textTheme.titleLarge?.copyWith(
+                fontWeight: FontWeight.bold,
+                fontSize: 20,
+              ),
+          children: [
+            TextSpan(
+              text: 'Who will be joining you?',
+              style: Theme.of(context).textTheme.titleLarge?.copyWith(
+                    fontWeight: FontWeight.normal,
+                    fontSize: 16,
+                  ),
+            ),
+          ],
+        ),
+      );
+  Widget continueButton(
+      {required GlobalKey<FormState> formKey, bool enabled = true}) {
+    return ElevatedButton(
+      onPressed: enabled
+          ? () {
+              if (formKey.currentState!.validate()) {
+                _createChildProfile();
+              }
+            }
+          : () {},
+      style: ElevatedButton.styleFrom(
+        backgroundColor: enabled ? AppTheme.givtLightGreen : Colors.grey,
+      ),
+      child: Text(
+        "Continue",
```

Comment:
I noticed that the ```'``` and the ```"``` are not consistent

Reply:
Yes, im not paying much attention to it since i still have to add all the strings to POEditor